src/pages/Settings/SettingsPage.vue

```diff
@@ -0,0 +1,224 @@
+						</li>
+						<li
+							v-if="isSubscriber"
+							class="section__subsection"
+						>
+							<h3>Auto lending settings</h3>
+							<p>Turn auto lending on or off, and put your idle credit to good use</p>
+						</li>
+						<li class="section__subsection">
+							<template v-if="isSubscriber">
+								<h3>Monthly Good settings</h3>
+								<p>Automatically support borrowers in need each month</p>
+							</template>
+							<template v-if="!isSubscriber">
+								<h3>Auto deposit</h3>
+								<p>Increase your impact - make regular deposits or donations into your Kiva account</p>
+							</template>
+						</li>
+					</ul>
+				</a>
+			</section>
+			<section class="section">
+				<a
+					class="section__link"
+					href="/settings/email"
+				>
+					<h2 class="section__title">Email preferences</h2>
+					<ul class="section__list">
+						<li>
+							<h3>Account and loan updates</h3>
+							<p>Receive borrower updates and repayment notifications</p>
+						</li>
+						<li>
+							<h3>News from Kiva</h3>
+							<p>Hear about what’s new at Kiva and how you can get more involved</p>
+						</li>
+						<li>
+							<h3>Lender and team messages</h3>
+							<p>Get notifications from other lenders and teams you belong to</p>
+						</li>
+					</ul>
+				</a>
+			</section>
+			<section class="section">
+				<a
+					class="section__link"
+					href="/trustees/apply"
+				>
+					<h2 class="section__title">Trustee settings</h2>
+				</a>
+			</section>
+			<section class="section">
+				<a
+					class="section__link"
+					href="/settings/data"
+				>
+					<h2 class="section__title">Data settings</h2>
+					<ul class="section__list">
+						<li>
+							<h3>Cookie preferences</h3>
+							<p>Change your Kiva cookie preferences</p>
+						</li>
+						<li>
+							<h3>Advertising data sharing preferences</h3>
+							<p>Change your Kiva data sharing preferences for advertising purposes</p>
+						</li>
+						<li>
+							<h3>Account termination</h3>
+							<p>Close your Kiva account</p>
+						</li>
+					</ul>
+				</a>
+			</section>
+		</div>
+	</www-page>
+</template>
+
+<script>
+import gql from 'graphql-tag';
+
+import TheMyKivaSecondaryMenu from '@/components/WwwFrame/Menus/TheMyKivaSecondaryMenu';
+import WwwPage from '@/components/WwwFrame/WwwPage';
+
+const pageQuery = gql`query settingsQuery {
+	my {
+		autoDeposit {
+			id
+			isSubscriber # signifies monthly good is active
+		}
+	}
+	general {
+		featureSetting(key: "mfa.enabled") {
+			key
+			value
+			description
+		}
+	}
+}`;
+
+export default {
+	components: {
+		TheMyKivaSecondaryMenu,
+		WwwPage,
+	},
+	metaInfo: {
+		title: 'Settings',
+	},
+	inject: ['apollo'],
+	data() {
+		return {
+			isMfaActive: false,
+			isSubscriber: false,
+		};
+	},
+	apollo: {
+		query: pageQuery,
+		prefetch: true,
+		result({ data }) {
+			this.isSubscriber = data?.my?.autoDeposit?.isSubscriber ?? false;
+			this.isMfaActive = data?.general?.featureSetting?.value ?? false;
```

Comment:
I think you'll need to use `settingsUtils.readBoolSetting` for this

Reply:
@emuvente Made a tweak to check against 'true' as a string, does that look like it'll work to you? Seems ok in my testing.

Reply:
👍